Documentation/Input/Gestures.md

```diff
@@ -0,0 +1,30 @@
+# Gestures
+
+Gestures are input events based on human hands. There are two types of devices that raise gesture input events in MRTK:
+- Windows Mixed Reality devices such as Hololens. This describes pinching motions ("Air Tap") and tap-and-hold gestures.
+
+  For more information on Hololens gestures see the [Windows Mixed Reality Gestures documentation](https://docs.microsoft.com/en-us/windows/mixed-reality/gestures).
+
+  [`WindowsMixedRealityDeviceManager`](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityDeviceManager) wraps the [Unity XR.WSA.Input.GestureRecognizer](https://docs.unity3d.com/ScriptReference/XR.WSA.Input.GestureRecognizer.html) to consume Unity's gesture events from Hololens devices.
+
+- Touch screen devices.
+
+  [`UnityTouchController`](xref:Microsoft.MixedReality.Toolkit.Input.UnityInput) wraps the [Unity Touch class](https://docs.unity3d.com/ScriptReference/Touch.html) that supports physical touch screens.
+
+Both of these input sources use the _Gesture Settings_ profile to translate Unity's Touch and Gesture events respectively into MRTK's [Input Actions](InputActions.md). This profile can be found under the _Input System Settings_ profile.
+
+<img src="../../External/ReadMeImages/Input/GestureProfile.png" style="max-width:100%;">
+
+Gesture events are received by implementing one of the gesture handler interfaces: [`IMixedRealityGestureHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityGestureHandler)  and [`IMixedRealityGestureHandler<TYPE>`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityGestureHandler`1) (see table of [event handlers](InputEvents.md)).
+When implementing the generic version, the *OnGestureCompleted* and *OnGestureUpdated* events can receive typed data of the following types:
+- ```Vector2```
+  2D position gesture. Currently supported sources:
+  - [```deltaPosition```](https://docs.unity3d.com/ScriptReference/Touch-deltaPosition.html) on a touch screen
+- ```Vector3```
+  3D position gesture. Currently supported sources:
+  - [```cumulativeDelta```](https://docs.unity3d.com/ScriptReference/XR.WSA.Input.ManipulationUpdatedEventArgs-cumulativeDelta.html) of a Hololens manipulation event
+  - [```normalizedOffset```](https://docs.unity3d.com/ScriptReference/XR.WSA.Input.NavigationUpdatedEventArgs-normalizedOffset.html) of a Hololens navigation event
+- ```Quaternion```
```

Comment:
I think they're there so users are aware that the input system can raise gesture events with Quaternion or Poses even thought currently there are no input sources that produce them. We could rephrase "currently supported" to something like "current produced by". Including @lukastoenneMS , who wrote that bit.

Reply:
Luis is correct, the event functions exist but will never get called atm, because there is no device device that actually raises them. Feel free to reword it.

Reply:
Okay, I'm going to resolve this one for now and we can look to improve some stuff going forward.